Index: rasdaemon-0.6.7/util/ras-mc-ctl.in
===================================================================
--- rasdaemon-0.6.7.orig/util/ras-mc-ctl.in
+++ rasdaemon-0.6.7/util/ras-mc-ctl.in
@@ -793,7 +793,9 @@ sub register_dimm_labels
     my $vendor = lc $conf{mainboard}{vendor};
     my $model  = lc $conf{mainboard}{model};
     my $pvendor = lc $conf{mainboard}{product_vendor};
-    my $pname = lc $conf{mainboard}{product_name};
+    my $pname = '';
+    $pname = lc $conf{mainboard}{product_name}
+        if ($conf{mainboard}{product_name});
     my $sysfs  = "/sys/devices/system/edac/mc";
 
     if (!exists $$lref{$vendor}{$model} && !exists $$lref_prod{$pvendor}{$pname}) {
